doc/faq.rst

```diff
@@ -385,3 +385,7 @@ efficient to process for most operations. Extensive work would also be needed
 to support Pandas categorical types. Restricting input to homogeneous
 types therefore reduces maintenance cost and encourages usage of efficient
 data structures.
+
+Do you plan to implement transform for target y in a pipeline? Currently transform only works for features X in a pipeline.
+----------------------------------------------------------------------------
+This a long-standing discussion about not being able to transform y in a pipeline, follow on github issue #4143. Meanwhile check out TargetTransformer, pipegraph, `imbalanced-learn<https://github.com/scikit-learn-contrib/imbalanced-learn>`
```

Comment:
1. Please make this <80 lines to match the convention in the file.

2. Are you refering to `TransformedTargetRegressor` here?

3. Link to pipegraph?

Reply:
@thomasjpfan thanks for reviewing.
1. Do you mean the title? Edited to just one line
2. Confirmed with amueller and changed to TransformedTargetRegressor 
3. Confirmed with amueller adn linked to pipegraph

Reply:
@thomasjpfan 
1. Fixed line length
2. Fixed. Style improved
3. Fixed